openreview/arr/process/preprint_release_submission_process.py

```diff
@@ -37,6 +51,46 @@ def post_submission_edit(submission):
             note=updated_note,
             signatures=[venue_id]
         )
+
+        paper_link = submission.content.get('previous_URL', {}).get('value')
+        # If previous submission, change reader set to include previous reviewers submitted group
+        if paper_link:
+            content = {
+                'explanation_of_revisions_PDF': {
+                    'readers': [
```

Comment:
instead of building the whole readers array, you can use the `append` keyword in the edit

Reply:
Using append and delete to replace the reviewers with reviewers submitted group